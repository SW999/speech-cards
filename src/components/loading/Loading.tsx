import * as React from 'react';
import { FunctionComponent } from 'react';

export const Loading: FunctionComponent = () => (
  <div className="loading">
    <h1>Loading ...</h1>
  </div>
);
