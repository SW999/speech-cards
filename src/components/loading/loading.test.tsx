import React from 'react';
import { render } from '@testing-library/react';
import Loading from './Loading';

describe('Loading component', () => {
  it('Loading should take a snapshot', () => {
    const { asFragment } = render(<Loading />);

    expect(asFragment()).toMatchSnapshot('Loading');
  });
});
