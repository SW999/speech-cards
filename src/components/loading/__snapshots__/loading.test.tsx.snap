// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component Loading should take a snapshot: Loading 1`] = `
<DocumentFragment>
  <div
    class="loading"
  >
    <h1>
      Loading ...
    </h1>
  </div>
</DocumentFragment>
`;
